draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -2782,88 +2776,62 @@ Sun Nov  6 08:49:37 1994         ; ANSI C's asctime() format
 
 <section title="Initiating HTTP Over TLS" anchor="https.initiation">
 <t>
-   Conceptually, HTTP/TLS is very simple. Simply use HTTP over TLS
-   precisely as you would use HTTP over TCP.
+   Conceptually, HTTP/TLS is simply sending HTTP messages over a connection
+   secured via TLS <xref target="RFC8446"/>.
 </t>
 <t>
-   The agent acting as the HTTP client should also act as the TLS
-   client.  It should initiate a connection to the server on the
-   appropriate port and then send the TLS ClientHello to begin the TLS
-   handshake. When the TLS handshake has finished. The client may then
-   initiate the first HTTP request.  All HTTP data MUST be sent as TLS
-   "application data".  Normal HTTP behavior, including retained
-   connections should be followed.
+   The HTTP client also acts as the TLS client. It initiates a connection to
+   the server on the appropriate port and sends the TLS ClientHello to begin
+   the TLS handshake. When the TLS handshake has finished, the client may then
+   initiate the first HTTP request. All HTTP data &MUST; be sent as TLS
+   "application data", but is otherwise treated like a normal connection for
+   HTTP (including potential reuse as a persistent connection).
```

Comment:
I think the best way forward is to move this section to Messaging and fix it there, rather than try to describe HTTP/1.1-specific connection initiation over TLS within Semantics. Then the certificate verification can be described immediately  after https origins.